package.json

```diff
@@ -55,6 +55,7 @@
     "@types/node": "~8.9.4",
     "codelyzer": "~4.5.0",
     "jest": "^23.6.0",
+    "jest-cli": "^24.5.0",
```

Comment:
Just checking, is this change needed for this patch to work? If not let's remove it.

Reply:
This is a change made by jest when updating the snapShot so it doesn't give an error during checks.

Reply:
@ritisha2000 I don't believe so. You should not need `jest-cli` in the first place. How are you running the tests?

Reply:
I assumed it was necessary since it wasn't added by me. I ran the "npm test" command it doesn't seem to show any failing tests without the line so I will remove it.